strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/io/ArtifactOutputStreamTest.java

```diff
@@ -42,13 +42,22 @@
 {
 
     public static final String REPOSITORY_RELEASES = "aos-releases";
+    
+    private static boolean initialized;
```

Comment:
Anyway, I'll re-open SB-757 and I'll be basing on your branch and will try to find out what's wrong.